src/content/en/2020/security.md

```diff
@@ -78,9 +75,11 @@ Whilst the high volume of requests is encouraging, these can often be dominated
   )
 }}
 
+Whilst the high volume of requests is encouraging, these can often be dominated by [third-party](./third-parties) requests and services like Google Analytics, fonts or advertisements. Websites themselves can lag, but again we see encouraging use with between 73% and 77% of sites now being served over HTTPS.
+
 ### Protocol versions
 
-As HTTPS is now well and truly the norm, the challenge moves from having any sort of HTTPS, to ensuring that secure versions of the underlying TLS (Transport Layer Security) protocol are being used. TLS needs maintenance as versions become older, vulnerabilities are found and compute increase making attacks more achievable.
+As HTTPS is now well and truly the norm, the challenge moves from having any sort of HTTPS, to ensuring that secure versions of the underlying TLS (Transport Layer Security) protocol are being used. TLS needs maintenance as versions become older, vulnerabilities are found and compute increases making attacks more achievable.
```

Comment:
Perhaps "computational power" instead of "compute"?

Reply:
```suggestion
As HTTPS is now well and truly the norm, the challenge moves from having any sort of HTTPS, to ensuring that secure versions of the underlying TLS (Transport Layer Security) protocol are being used. TLS needs maintenance as versions become older, vulnerabilities are found and compute power increases making attacks more achievable.
```